navigator_helpers/tasks/prompt_templates/sample_to_dataset.py

```diff
@@ -301,3 +300,33 @@
 Reread the instructions before proceeding, paying attention to format
 in examples: {data_generation_prompt}
 """
+
+DATASET_SCHEMA_PREPROCESSING_PROMPT_TEMPLATE = """
+You are an expert data practioner, skilled in critically evaluating data and
+leveraging expertise across various domains, including data analysis, data
+engineering, and a general understanding of programming concepts.
+
+Examine a list of column names provided in the <schema></schema> tags below.
+<schema>
+{sampled_dataset_column_list}
+</schema>
+
+Fix formatting of these column names so that they:
+1. appear in lower case. Example: "FIRST_name" -> "first_name"
```

Comment:
Why get the LLM to do this, potentially with errors, rather than lower, snake case and remove non-AN characters from the text programmatically?  I imagine we could also swap the first two words in the case that the first parses to a number as well.